go.mod

```diff
@@ -276,6 +276,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/opencontainers/image-spec v1.1.0-rc5 // indirect
 	github.com/opentracing/opentracing-go v1.2.0 // indirect
+	github.com/package-url/packageurl-go v0.1.2 // indirect
```

Comment:
It's odd to me that this was added with ec-cli at `4a1afa30abc7`, we needed at least `f9f46907` for it to be added. So I've ran `go mod tidy` on this branch and this line was removed. Are we missing a version bump of ec-cli?

Should this include changes done by running `go get -u github.com/enterprise-contract/ec-cli`?

Reply:
Actually, I don't see purl being used here, perhaps that doesn't belong to this PR?

Reply:
A rebase seems to fix it. Should just be changes to `cve.rego` and `cve_test.rego` now.